Source/MultipartFormData.swift

```diff
@@ -266,6 +266,29 @@ public class MultipartFormData {
     }
 
     /**
+    Creates a body part from the data and appends it to the multipart form data object.
+    This is to upload non-file data, e.g. a JSON string from the form input
+    
+    The body part data will be encoded using the following format:
+    
+    - `Content-Disposition: form-data; name=#{name};mimeType=#{mimeType}` (HTTP Header)
+    - Encoded data
+    - Multipart form boundary
+    
+    :param: data The data to encode into the multipart form data.
+    :param: name The name to associate with the data in the `Content-Disposition` HTTP header.
+    :param: mimeType The MIME type to associate with the data in the `Content-Type` HTTP header.
+    */
+    public func appendBodyPart(#data: NSData, name: String, mimeType: String) {
```

Comment:
I was actually thinking the same thing @kylef. The more I think about it though, wouldn't you always want to provide a filename as a hint to the server? We could certainly make the `fileName` parameter optional, but is there really a realistic case where that is what the client should do?